staging/src/k8s.io/client-go/discovery/cached/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cached provides a discovery client for memcached.
```

Comment:
Package cached provides an discovery client with in-memory cache.

Reply:
memcached?

Reply:
@noqcks Can you address @yue9944882's comment above?

Reply:
My bad, I already updated this line according to @yue9944882 comment but didn't reply.

> // Package cached provides a discovery client with in-memory cache.